ProcessLib/ProcessVariable.h

```diff
@@ -60,8 +60,9 @@ class ProcessVariable
 	MeshLib::Mesh const& getMesh() const;
 
 	void initializeDirichletBCs(MeshGeoToolsLib::MeshNodeSearcher& searcher,
-	                            std::vector<std::size_t>& global_ids,
-	                            std::vector<double>& values);
+	        const AssemblerLib::LocalToGlobalIndexMap& local2global,
+	        const unsigned nodal_dof_idx,
+	        std::vector<std::size_t>& global_ids, std::vector<double>& values);
```

Comment:
When you merge or rebase, you need to change ``td::size_t`` here to ``GlobalIndexType``.